device/juniper/x86_64-juniper_qfx5200-r0/plugins/qfx5200_eeprom_data.py

```diff
@@ -56,11 +51,9 @@ def fantype_detect():
     for filename in os.listdir(refpgaTMC_path):
         if filename.endswith('_type'):
             fantype_path = os.path.join(refpgaTMC_path, filename)
-            cat_string = "cat "
-            fantype_string = cat_string + fantype_path
-            status, fan_type = commands.getstatusoutput(fantype_string)
-            if ((fan_type == AFO) or (fan_type == AFI)):
-                return fan_type
+            p = subprocess.run(["cat", fantype_path])
```

Comment:
>run [](http://example.com/codeflow?start=27&length=3)

Do you need to provide optional parameter `capture_output`?

Reply:
You're right, I added `capture_output`, thanks!